specifyweb/frontend/js_src/lib/components/QueryBuilder/Line.tsx

```diff
@@ -372,6 +373,26 @@ export function QueryLine({
                 mappingElementDivider
               )
             )}
+            {(fieldMeta.fieldType === 'formatter' ||
+              fieldMeta.fieldType === 'aggregator') &&
+            typeof fieldMeta.tableName === 'string' ? (
```

Comment:
I would say no, we try in our code to not use && if there is more than one element to check, here we have 3.

Reply:
yes, that's a rule I try to follow as it makes things easier to reason about

i.e if you have a simple case like `condition && JSX`, then it's clear that `&&` is a sort of conditional render 
but when you have complex condition like `condition1 || condition2 && JSX`, not only is it harder to reason about the order, but you could also make mistakes if you forget to add parenthesis - with ternary, this is much more explicit and readable

same in case of `condition1 && condition2 && condition3 && JSX` - having `condition1 && condition2 && condition3 ? JSX : undefined` is more readable, even if a bit longer